### PR TITLE
merge progressive myoclonic epilepsy with neuroserpin inclusion bodies

### LINKED ISSUE #6406: [Merge] MONDO:0035160 'progressive myoclonic epilepsy with neuroserpin inclusion bodies'
**Mondo term (ID and Label)**
MONDO:0035160 'progressive myoclonic epilepsy with neuroserpin inclusion bodies'

**Reason for deprecation**
duplicate

**Suggested term to merge**
MONDO:0012391 'neuronal ceroid lipofuscinosis 8 northern epilepsy variant'

discussed with Ada on 2023-06-28

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #6406

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -245209,17 +245210,21 @@ synonym: "progressive epilepsy - intellectual disability, Finnish type" RELATED
 synonym: "progressive epilepsy with intellectual disability, northern epilepsy" EXACT [DOID:0110724]
 synonym: "progressive epilepsy with mental retardation, northern epilepsy" EXACT DEPRECATED [DOID:0110724]
 synonym: "progressive epilepsy-intellectual disability syndrome, Finnish type" EXACT [DOID:0110724, GARD:0004010]
+synonym: "progressive myoclonic epilepsy with neuroserpin inclusion bodies" EXACT [MONDO:0035160]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:245213]: Where did this synonym come from? I don't see it on `MONDO:0035160`?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -245188,14 +245188,15 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/9946/arthrog
 id: MONDO:0012391
 name: neuronal ceroid lipofuscinosis 8 northern epilepsy variant
 def: "Progressive epilepsy-intellectual deficit, Finnish type (also known as Northern epilepsy) is a subtype of neuronal ceroid lipofuscinosis (NCL) characterized by seizures, progressive decline of intellectual capacities and variable loss of vision." [Orphanet:1947]
-subset: gard_rare
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:245191]: Why is this removed?

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -245209,17 +245210,21 @@ synonym: "progressive epilepsy - intellectual disability, Finnish type" RELATED
 synonym: "progressive epilepsy with intellectual disability, northern epilepsy" EXACT [DOID:0110724]
 synonym: "progressive epilepsy with mental retardation, northern epilepsy" EXACT DEPRECATED [DOID:0110724]
 synonym: "progressive epilepsy-intellectual disability syndrome, Finnish type" EXACT [DOID:0110724, GARD:0004010]
+synonym: "progressive myoclonic epilepsy with neuroserpin inclusion bodies" EXACT [MONDO:0035160]
 xref: DOID:0110724 {source="MONDO:equivalentTo"}
 xref: ICD10CM:E75.4 {source="DOID:0110724", source="Orphanet:1947/attributed", source="Orphanet:1947/ntbt", source="Orphanet:1947"}
+xref: ICD10CM:G40.3 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:530298"}
 xref: OMIM:610003 {source="Orphanet:1947/e", source="DOID:0110724", source="MONDO:equivalentTo", source="Orphanet:1947"}
 xref: Orphanet:1947 {source="OMIM:610003", source="DOID:0110724", source="MONDO:equivalentTo", source="GARD:0004010"}
+xref: Orphanet:530298
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:245219]: This is missing `{source="MONDO:equivalentTo"}` from `MONDO:0035160`. Was this dropped using "owltools"... is this an existing bug?

**Review by twhetzel — CHANGES_REQUESTED** [review]
See comments inline.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:245191]: it is a duplicate, it's another external file too (there is an external mondo-subsets.owl file)

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:245213]: that was the name/label of the term that we merged in

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:245219]: I purposely removed it so we didn't have a proxy merge. Ada advised me to merge these terms but Orphanet has not merged them.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:245213]: Oh right! I need to update the [docs](https://mondo.readthedocs.io/en/latest/editors-guide/merging-and-obsoleting/) to include that obvious statement.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:245191]: Ok. What is it a duplicate of?

**Review by twhetzel — APPROVED** [review]

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:245191]: the exact same annotation is in the mondo-subsets.owl file.

I think these were moved into the external file when @sabrina and @matentzn created the new rare disease subsets.

See: https://github.com/monarch-initiative/mondo/issues/6167

Comment (twhetzel) [reply src/ontology/mondo-edit.obo:245191]: Ok, I don't know what the `mondo-subsets.owl` file is.

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:245191]: it is in mondo/components

it looks like we don't have any documentation on it, can you bring this up on the tech call? Nico can explain what it is, and we should create some documentation.

Comment (twhetzel) [reply src/ontology/mondo-edit.obo:245191]: Thank you! I added this to the tech call agenda.
